client/state/action-types.js

```diff
@@ -23,6 +23,9 @@ export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
 export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
 export const FETCH_WPORG_PLUGIN_DATA = 'FETCH_WPORG_PLUGIN_DATA';
 export const NEW_NOTICE = 'NEW_NOTICE';
+export const POST_REQUEST = 'POST_REQUEST';
+export const POST_REQUEST_SUCCESS = 'POST_REQUEST_SUCCESS';
+export const POST_REQUEST_FAILURE = 'POST_REQUEST_FAILURE';
```

Comment:
(note: not really for this PR)

I wonder if we should flip all of these to keymirror... It turns out the const doesn't transfer through the export, so an importer can still reassign the imported variable. I feel like the `const` is giving a false sense of security. 

But maybe it helps with code completion in some editors?